lib/ioki/apis/platform_api.rb

```diff
@@ -123,6 +123,12 @@ class PlatformApi
         except:      [:create, :update, :delete],
         model_class: Ioki::Model::Platform::TaskList
       ),
+      Endpoints.custom_endpoints(
+        'task_list',
```

Comment:
Just some nit-picking here. I think we use the plural form of the model for custom endpoints. But I have no idea if this is really a rule or common practice 🤷 

No hard opinion here :)